widgetastic_manageiq/__init__.py

```diff
@@ -5084,3 +5084,46 @@ class MigrationDashboardStatusCard(AggregateStatusCard):
         ' h2[contains(@class,"card-pf-title")  and contains(.,{@name|quote})]]'
     )
     TITLE_ANCHOR = ".//h2"
+
+
+class MonitorStatusCard(ParametrizedView):
```

Comment:
@digitronik This is the `MonitorStatusCard` widget. Here I am trying to mirror the behavior of `ParametrizedStatusBox`, only instead of using `StatusBox` for the card, I am using `MonitorCard`